data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -350,18 +284,16 @@ else if(nodeSelExpr.getChildCount()==2 || nodeSelExpr.getChildCount()==1 && node
 			Relation relation = new Relation(conNumeric, null);
 			relations.add(relation);
 		} catch (NumberFormatException e) {
-			System.out.println("Unrecognizable column type");
+			LOGGER.info("Unrecognizable column type");
 		}
 		}
 		else
 			col = colName = nodeSelExpr.getChild(0).getChild(0).toString();
 if(col != null && colName != null) {
-	System.out.println("Input column = " + col);
+	LOGGER.info("Input column = " + col);
 	columnList.addToList("input->" + col);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "input->" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)